apps/functions/dns-redirecting/index.ts

```diff
@@ -46,6 +46,15 @@ export const dnsRedirecting = functions.https.onRequest(
     if (hostname === 'blog.angular.io') {
       response.redirect(redirectType, `https://blog.angular.dev${request.originalUrl}`);
     }
+    if (hostname === 'material.angular.io') {
+      response.redirect(redirectType, `https://material.angular.dev${request.originalUrl}`);
+    }
+    if (hostname.endsWith('material.angular.io')) {
```

Comment:
what would be some of these sub-sub-domains?

Reply:
Ah, didn't even think about that. SGTM (nit: might want to try `endsWith('.material...')`